[2025-03-22T09:56:51.386] [INFO] sn - Logger initialized for sn
[2025-03-22T09:56:51.387] [INFO] database - Database: sn connection pool initialized
[2025-03-22T09:56:51.389] [INFO] data-parser - snParser init
[2025-03-22T09:56:51.551] [INFO] data-parser - Found 308 rows to process
[2025-03-22T09:56:51.629] [ERROR] data-parser - snParser error: Error: Unknown column 'orzeczenie_id' in 'field list'
    at PromisePoolConnection.query (C:\Users\przyg\Desktop\GitHub\ankork-quickmap\node_modules\mysql2\lib\promise\connection.js:29:22)
    at saveDataToDatabase (C:\Users\przyg\Desktop\GitHub\ankork-quickmap\src\services\parsers\snParser.js:78:30)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async snParser (C:\Users\przyg\Desktop\GitHub\ankork-quickmap\src\services\parsers\snParser.js:183:13)
    at async DataParser.sn [as parsingStrategy] (C:\Users\przyg\Desktop\GitHub\ankork-quickmap\src\services\dataParser.js:26:16)
    at async DataParser.parse (C:\Users\przyg\Desktop\GitHub\ankork-quickmap\src\services\dataParser.js:15:16)
    at async Promise.all (index 0)
    at async Object.scraper (C:\Users\przyg\Desktop\GitHub\ankork-quickmap\src\models\sn\sn.js:183:5) {
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sql: 'INSERT IGNORE INTO orzeczenia_sedziowie (orzeczenie_id, sedzia_id) VALUES (1, 1)',
  sqlState: '42S22',
  sqlMessage: "Unknown column 'orzeczenie_id' in 'field list'"
}
